bom/pom.xml

```diff
@@ -153,6 +153,11 @@
         <artifactId>vertx-micrometer-metrics</artifactId>
         <version>${vertx.version}</version>
       </dependency>
+      <dependency>
```

Comment:
We'll need to sort out all CQs for this PR

Reply:
No more CQs :D as long as the license is ok.